src/sbvr-api/sbvr-utils.ts

```diff
@@ -1386,8 +1387,16 @@ const runODataRequest = (req: Express.Request, vocabulary: string) => {
 
 				let request: uriParser.ODataRequest | uriParser.ODataRequest[];
 				if (Array.isArray(parsedRequest)) {
+					if (parsedRequest.some(isActionRequest)) {
```

Comment:
I suppose it is not too hard to add support for it inside $batch by passing a tx to `runAction` which is then shared across all requests in the $batch but I would rather start without it considering we don't even properly support $batch